vscode/src/VsCodeExtension.ts

```diff
@@ -48,6 +48,100 @@ export class VsCodeExtension {
       setupWebviewMessageListener(webview, this.state);
     });
 
+    // // DEBUG USE ONLY
```

Comment:
Should just delete this blob right? Any reason to keep it?

Reply:
It's pretty useful for webview development. Maybe we can keep until we have time to implement something else like MSW for mock data.